components/script/dom/htmlinputelement.rs

```diff
@@ -1228,3 +1230,8 @@ impl Activatable for HTMLInputElement {
         }
     }
 }
+
+fn filter_from_accept(_s: DOMString) -> Vec<FilterPattern> {
+    unimplemented!()
```

Comment:
Nice!

So that this can merge, though, could we still use the empty vec so that it doesn't crash and can be used?